frontend/model/contracts/group.js

```diff
@@ -120,6 +120,15 @@ function memberLeaves ({ username, dateLeft }, { meta, state, getters }) {
   updateCurrentDistribution({ meta, state, getters })
 }
 
+function isActionYoungerThanUser (actionMeta: Object, userProfile: ?Object): boolean {
+  // A util function that checks if an action (or event) in a group occurred after a particular user joined a group.
+  // This is used mostly for checking if a notification should be sent for that user or not.
+  // e.g.) user-2 who joined a group later than user-1 (who is the creator of the group) doesn't need to receive
+  // 'MEMBER_ADDED' notification for user-1.
+  return Boolean(userProfile) &&
```

Comment:
I think `Boolean(userProfile) &&` can be removed and the type annotation above modified to `Object` instead of `?Object`.

Anywhere in the calling function where myProfile/userProfile is null, that's where the check can be added if necessary. And as I'm looking through the code, I don't see where that is even necessary.